include/ebpf_extension.h

```diff
@@ -91,11 +91,21 @@ typedef struct _ebpf_extension_program_dispatch_table
     ebpf_program_batch_end_invoke_function_t ebpf_program_batch_end_invoke_function;
 } ebpf_extension_program_dispatch_table_t;
 
-typedef struct _ebpf_extension_data
+typedef struct _ebpf_extension_data_1
```

Comment:
```suggestion
typedef struct _ebpf_extension_data_v1
```
I recommend changing `_1` to `_v1` and same for `_2`